Source/WebCore/html/HTMLInputElement.cpp

```diff
@@ -2373,4 +2373,10 @@ float HTMLInputElement::switchAnimationPressedProgress() const
     return downcast<CheckboxInputType>(*m_inputType).switchAnimationPressedProgress();
 }
 
+bool HTMLInputElement::isSwitchPressed() const
```

Comment:
I don't think this method is called anywhere?

Reply:
This is needed by the internal drawing code.

I was thinking that maybe we want to put this behind IOS_TOUCH_EVENTS as well for now to reduce impact on other platforms.

Reply:
I think we should guard it somehow, otherwise there's nothing stopping someone from removing it.

Reply:
I also wonder if this should be `isSwitchHeld`? The switch is technically pressed before the timer fires.